website/docs/r/networkfirewall_resource_policy.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Network Firewall"
+layout: "aws"
+page_title: "AWS: aws_networkfirewall_resource_policy"
+description: |-
+  Provides an AWS Network Firewall Resource Policy resource.
+---
+
+# Resource: aws_networkfirewall_resource_policy
+
+Provides an AWS Network Firewall Resource Policy Resource for a rule group or firewall policy.
+
+## Example Usage
+
+### For a Firewall Policy resource
+
+```hcl
+resource "aws_networkfirewall_resource_policy" "example" {
+  resource_arn = aws_networkfirewall_firewall_policy.example.arn
+  policy = jsonencode({
+    Statement = [{
+      Action   = "network-firewall:ListFirewallPolicies"
+      Effect   = "Allow"
+      Resource = aws_networkfirewall_firewall_policy.example.arn
+      Principal = {
+        AWS = "arn:aws:iam::1234567890:user/example"
```

Comment:
```suggestion
        AWS = "arn:aws:iam::123456789012:root"
```

Reply:
ahh right that should match what I used in the acctests 👍